nsds_lab_to_nwb/components/neural_data/neural_data_originator.py

```diff
@@ -1,31 +1,40 @@
 import logging.config
-from nsds_lab_to_nwb.components.htk.htk_manager import HTKManager
-from nsds_lab_to_nwb.components.tdt.tdt_manager import TDTManager
+from pynwb.ecephys import ElectricalSeries
+
+from nsds_lab_to_nwb.components.htk.htk_reader import HTKReader
+from nsds_lab_to_nwb.components.tdt.tdt_reader import TDTReader
+
 
 logger = logging.getLogger(__name__)
 
 
 class NeuralDataOriginator():
-    def __init__(self, dataset, metadata, use_htk=False):
+    def __init__(self, dataset, metadata):
         self.dataset = dataset      # this should have all relavant paths
         self.metadata = metadata    # this should have all relevant metadata
 
-        if use_htk:
+        if hasattr(self.dataset, 'htk_mark_path'):
             logger.info('Using HTK')
-            self.neural_data_manager = HTKManager(self.dataset.htk_path)
+            self.neural_data_reader = HTKReader(self.dataset.htk_path)
         else:
             logger.info('Using TDT')
-            self.neural_data_manager = TDTManager(self.dataset.tdt_path)
+            self.neural_data_reader = TDTReader(self.dataset.tdt_path)
 
     def make(self, nwb_content, electrode_table_regions):
         for device_name, dev_conf in self.metadata['device'].items():
             if isinstance(dev_conf, str): # skip other annotations
                 continue
 
-            e_series = self.neural_data_manager.extract(device_name, dev_conf,
-                                                        electrode_table_regions[device_name])
-            if e_series is None:
-                logger.info('No e-series extracted. Skipping...')
+            data, metadata = self.neural_data_reader.get_data(device_name, dev_conf)
+            if data is None:
+                logger.info(f'No data availble for {device_name}. Skipping...')
             else:
-                logger.info('Adding extracted e-series to NWB...')
+                electrode_table_region = electrode_table_regions[device_name]
+                e_series = ElectricalSeries(name=device_name,
+                                            data=data,
+                                            electrodes=electrode_table_region,
+                                            starting_time=0.,
```

Comment:
Should `starting_time` be 0, or is it the `utc_start_time` metadata extracted from TDT?

Reply:
I'm not sure. I'll make an issue to check that times are all correct.